packages/teams-js/webpack.config.js

```diff
@@ -52,5 +62,19 @@ module.exports = {
       removeSource: true,
       outputAsModuleFolder: true,
     }),
+    new HtmlWebpackPlugin(),
+    new SubresourceIntegrityPlugin({
+      enabled: true,
+    }),
+    new WebpackAssetsManifest({ integrity: true }),
+    {
+      apply: compiler => {
+        compiler.hooks.done.tap('wsi-test', () => {
+          const manifest = JSON.parse(readFileSync(join(__dirname, 'dist/assets-manifest.json'), 'utf-8'));
+          expect(manifest['MicrosoftTeams.js'].integrity).toMatch(/sha256-.*/);
```

Comment:
Is there a reason why we're using "sha256" here? To match what Teams v1 does, we should be generating sha384 hashes. I think I've seen in other discussions with you that this might not be a problem here, but I just wanted to call it out in case there is an impact here.